src/components/ComponentSettings/components/Circular/components/ValueMode/ValueMode.tsx

```diff
@@ -1,12 +1,12 @@
 import React, { FC } from 'react';
 import { RadioSelect } from '@keen.io/ui-core';
 
-import Row from '../Row';
-import Label from '../Label';
+import Row from '../../../../../Row';
+import Label from '../../../../../Label';
```

Comment:
pretty long path, what do you think about refactoring it later with webpack aliases? @maciejrybaniec @mczapkowicz @dariuszlacheta

Reply:
Yeah, webpack aliases definitely would be helpful :)

Reply:
Yes, @apryka could You please add ticket to our backlog - so we can discuss about it during refinement. 
If we are going to introduce Webpack aliases we need to think about them also in context of other front-end projects.